index.bs

```diff
@@ -1081,6 +1115,104 @@ The <dfn attribute for="XRInputSource">targetRaySpace</dfn> attribute is an {{XR
 The <dfn attribute for="XRInputSource">gripSpace</dfn> attribute is an {{XRSpace}} that tracks the pose that should be used to render virtual objects such that they appear to be held in the user's hand. If the user were to hold a straight rod, this {{XRSpace}} places the origin at the centroid of their curled fingers and where the <code>-Z</code> axis points along the length of the rod towards their thumb. The <code>X</code> axis is perpendicular to the back of the hand being described, with back of the users right hand pointing towards <code>+X</code> and the back of the user's left hand pointing towards <code>-X</code>. The <code>Y</code> axis is implied by the relationship between the <code>X</code> and <code>Z</code> axis, with <code>+Y</code> roughly pointing in the direction of the user's arm.
 
 The {{gripSpace}} MUST be <code>null</code> if the input source isn't inherently trackable such as for input sources with a {{targetRayMode}} of {{XRTargetRayMode/gaze}} or {{XRTargetRayMode/screen}}.
+
+The <dfn attribute for="XRInputSource">gamepad</dfn> attribute is a {{Gamepad}} that describes the state of any buttons and axes on the input device. If the {{XRInputSource}} does not represent a device with either multiple buttons or at least one axis the {{gamepad}} attribute MUST be <code>null</code>. 
```

Comment:
> either multiple buttons

Is the multiple buttons thing supposed to differentiate it from a mouse? If so ...uh... mice often have more than one button.  Is it explicitly excluding the HoloLens clicker?  What about cardboard?

Reply:
Mostly explicitly excluding things like the Cardboard button or HoloLens clicker while leaving room for something like the GearVR touchpad.

Reply:
I believe so, because this statement limits what should be exposed as a gamepad, as opposed to what is exposed as an `xr-standard` gamepad. I just don't see it being useful to expose "gamepads" for something that is functionally a binary button. The implication is that if the `gamepad` attribute is non-null you know that it means there's at least some level of more advanced input accessible.

I'm willing to reassess that if we feel it's not productive.

Reply:
Ah, I see what you're getting at.  Are you saying that an `XRInputSource` with a `targetRayMode` of 'tracked-pointer` that is a physical object with only a single button would simply fire "select" rather than have a gamepad object.  However, how would someone poll the value of the button if it is capable of reporting analog data?

Reply:
Yes, that was exactly the intent. Thanks for pointing out the analog button case, though. That *should* be a case where a gamepad is appropriate. Update the PR to account for that and added a non-normative note to explain the single-button select mechanic.